sxg_rs/src/crypto.rs

```diff
@@ -139,6 +139,33 @@ impl EcPrivateKey {
     }
 }
 
+// We have to implement serialization by ourself, because the macro `#[derive(Serialize)]`
+// can not produce the field ordering we need.
```

Comment:
Make a note here what the default serialize does that's not acceptable

Reply:
Done